docs/architecture/assets.md

```diff
@@ -1,58 +1,81 @@
+3. **Censorship resistance:**  
+   Users can transact freely and anonymously with no single contract or entity controlling asset transfers. This reduces the risk of censored transactions, resulting in a more open and resilient system.
 
-Having the issuer's ID encoded in the asset makes determining the type of an asset, inside and outside Miden VM, cost-efficient. And, representing the asset in a `word` means the representation is a commitment to the asset data itself. That is particularly interesting for non-fungible assets.
+4. **Flexible fee payment:**  
+   Unlike protocols that require a specific base asset for fees, Miden allows users to pay fees in any supported asset. This flexibility simplifies the user experience.
 
-### Issuance
+## What is an asset?
+
+An asset in Miden is a unit of value that can be transferred from one [account](accounts.md) to another using [notes](notes.md).
+
+Every asset is encoded using a single `Word` (4 field elements). This `Word` includes the [ID](accounts.md#id) of the issuing account and the asset details.
+
+
+## Native asset
+
+> All data structures following the Miden asset model that can be exchanged.
 
-Only specialized accounts called faucets can issue assets. As with regular accounts, anyone can create a faucet account. Faucets can issue either fungible or non-fungible assets - but not both. 
+Native assets adhere to the Miden asset model (encoding, issuance, storage). Every native asset is encoded using a single `Word` (4 field elements). This `Word` includes both the [ID](accounts.md#id) of the issuing account and the asset details.
 
-The `faucet_id` identifies the faucet and starts with a different sequence depending on the asset type, see the [account id discussion](accounts.md#account-id). The faucet's code defines rules for how assets can be minted, who can mint them etc. Conceptually, faucet accounts on Miden are similar to ERC20 contracts on Ethereum. However, there is no ownership tracking in Miden faucets.
+### Issuance
+
+> **Info**
+> - Only [faucet](accounts.md#account-type) accounts can issue assets.
 
-Faucets can create assets and immediately distribute them by producing notes. However, assets can also stay in the faucet after creation to be sent later, e.g., in a bundle. That way, one can mint a million NFTs locally in a single transaction and then send them out as needed in separate transactions in the future.
+Faucets can issue either fungible or non-fungible assets as defined at account creation. The faucet's code specifies the asset minting conditions: i.e., how, when, and by whom these assets can be minted. Once minted, they can be transferred to other accounts using notes.
 
 ![Architecture core concepts](../img/architecture/asset/asset-issuance.png)
```

Comment:
I think this picture could be improved:

- It feels disproportionally big (I think we can reduce the size of text and shapes quite a bit).
- I would probably not introduce new colors - e.g., in Miden VM docs arrows also use back color.
- I would maybe add some simple comments to make it easier to understand what's what.